docker/development/docker-compose.yml

```
@@ -0,0 +1,13 @@
+version: "3.8"
+
+include:
+  - supabase/docker/docker-compose.yml
+
+services:
+  supabase:
+    build:
+      context: ../../
+      dockerfile: ./Dockerfile
```

When running the commands in `README`, I got an error until I changed this to `./docker/development/Dockerfile` - I think it's because the path is relative to the context dir
```suggestion
      dockerfile: ./docker/development/Dockerfile
```